notebooks/working/6.0-mab-estimate-causal-effect.md

```diff
@@ -1825,32 +1825,210 @@ mnl_model_sim.fit_mle(np.zeros(num_vars),
+                                                  DTW_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  long_sim_data_causal.loc[long_sim_data_causal['mode_id']==5,'total_travel_time'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = DTW_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==5].shape[0]))
+```
+
+### Walk-Transit-Drive
+
+```python
+long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6,'total_travel_cost'] = \
+                                                 (WTD_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  WTD_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6,'total_travel_time'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = WTD_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6].shape[0]))
+```
+
+### Walk 
+
+
+No further change of variables since travel distance directly affects the utility.
+
+
+### Bike 
+
+
+No further change of variables since travel distance directly affects the utility.
+
+```python
+estimated_probabilities = mnl_model_sim.predict(long_sim_data_causal)
+```
+
+```python
+long_sim_data_causal['estimated_probabilities'] = estimated_probabilities
+```
+
+## Based on True Causal Graph and Original Estimated Model
+
+```python
+true_probabilities = mnl_model.predict(long_sim_data_causal)
+```
+
+```python
+long_sim_data_causal['true_probabilities'] = true_probabilities
+```
+
+### Plots
+
+```python
+long_sim_data['initial_probabilities'] = initial_probabilities
+```
+
+```python
+naive_effect = long_sim_data_naive.loc[long_sim_data_naive['mode_id']==1,'naive_probabilities'] - long_sim_data.loc[long_sim_data['mode_id']==1,'initial_probabilities']
+```
+
+```python
+estimated_effect = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==1,'estimated_probabilities'] - long_sim_data.loc[long_sim_data['mode_id']==1,'initial_probabilities']
+```
+
+```python
+true_effect = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==1,'true_probabilities'] - long_sim_data.loc[long_sim_data['mode_id']==1,'initial_probabilities']
+```
+
+```python
+plt.figure(figsize=(15,10))
+sns.distplot(true_effect, label ='True Effect')
+sns.distplot(naive_effect,label ='Naive Effect')
+plt.legend()
+```
+
```

Comment:
I've only really skipped to the end of your notebook to check out the plots but they're looking right!!

Can you add the following as it may be slightly easier to see than the plot itself?
```suggestion
all_effects =\
    pd.DataFrame({'true_effect': true_effect, 'naive_effect': naive_effect, 'estimated_effect': estimated_effect})
all_effects.describe()
```